membership_policy.md

```diff
@@ -117,6 +117,8 @@ The following apply to the part of project for which one would be an approver in
   * Nominated by an approver
     * With no objections from other approvers
     * Done through PR to update the top-level OWNERS file
+    
+In exceptional circumstances, such as when a repo is left without suffcient approvers, the project maintainers can pass a majority vote to add approvers that do not fulfil the requirements.
```

Comment:
IMO the bar is low enough as it is. Why should we accept exceptions?
Perhaps we can focus on providing SIG approvers the ability to nominate repo subproject approvers, that sounds a bit more loose.

Reply:
Well, if a project has no active maintainers, the "council" should either find one or archive the project.
I am more concerned that this 1% of the cases which you aim here will become in some projects a common norm.
But fine, I understand better from where you come with this.